installer/tools/pygobject_installer.ps1

```diff
@@ -14,30 +27,42 @@ if (-not $CacheDIR) {
 $pkgConfigLiteAppId = "bloodrock.pkg-config-lite"
 
 # Check if pkg-config-lite is already installed using winget
-$installedPackage = winget show --id $pkgConfigLiteAppId -e
-
-if ($installedPackage) {
-    Write-Output "Uninstalling existing $pkgConfigLiteAppId..."
-    winget uninstall --id $pkgConfigLiteAppId
+try {
+    $installedPackage = winget show --id $pkgConfigLiteAppId -e --accept-source-agreements
+    if ($installedPackage) {
+        Write-Output "Uninstalling existing $pkgConfigLiteAppId..."
+        winget uninstall --id $pkgConfigLiteAppId --accept-source-agreements
+    }
+} catch {
```

Comment:
can you use $_ in the catch block to provide more detailed error information?

catch {
    Write-Error "Failed to check/uninstall existing pkg-config-lite: $_"
    exit 1
}

Reply:
Updated in a114763